cookie-banner-rules-list.json

```diff
@@ -292,25 +292,33 @@
       "domains": ["azure.microsoft.com"]
     },
     {
-      "click": {
-        "optIn": ".js-accept-cookies",
-        "presence": ".js-consent-banner"
-      },
+      "id": "71443ce9-15b8-4e07-b51b-1f2158521ea9",
+      "domains": [
+        "askubuntu.com",
+        "mathoverflow.net",
+        "serverfault.com",
+        "stackapps.com",
+        "stackexchange.com",
+        "stackoverflow.blog",
+        "stackoverflow.co",
+        "stackoverflow.com",
+        "stackoverflowteams.com",
+        "superuser.com"
+      ],
       "cookies": {
         "optOut": [
           {
             "name": "OptanonAlertBoxClosed",
-            "value": "2023-01-04T15:56:30.908Z"
+            "value": "2030-01-01T00:00:00.000Z",
+            "sameSite": 1
```

Comment:
I don't think passing an explicit value for sameSite is needed? In my tests it worked fine without it.

Reply:
Thanks! Let's remove it so our code falls back to the default sameSite setting.

Reply:
Done. However, SameSite=Lax will soon become the default unless Mozilla's plans change in this regard (or have already changed).

Reply:
Done. However, SameSite=Lax will soon become the default value in Firefox unless Mozilla's plans change in this regard (or have already changed).

Reply:
Done. It's worth noting that SameSite=Lax will soon become the default value in Firefox unless Mozilla's plans change in this regard (or have already changed).